tests/admin_views/test_nav_sidebar.py

```diff
@@ -93,7 +93,7 @@ def test_sidebar_aria_current_page_missing_without_request_context_processor(sel
         )
         # Does not include aria-current attribute.
         self.assertContains(response, '<a href="%s">Users</a>' % url)
-        self.assertNotContains(response, "aria-current")
+        self.assertNotContains(response, '<a href="%s" aria-current="page">Users</a>' % url)
```

Comment:
I'm not sure it's really necessary to change this test? Does it fail without being changed?

If we do need this, then it can be rewrapped:

```suggestion
        self.assertNotContains(
            response, f'<a href="{url}" aria-current="page">Users</a>'
        )
```